### ISSUE TITLE
MVI Install uses wrong NVIDIA driver on IBM Cloud

### ISSUE THREAD

**Issue by delpiccolod** [ibm-mas/ansible-devops#1511]
### Collection version

MAS CLI

### Environment information

```console
quay.io/ibmmas/cli:v6-241003-amd64
```

### What happened?

Installing MVI via the CLI on a brand new cluster on IBM Cloud caused a failure during the NVIDIA driver installation. The version that was being installed by default was: 

driver:
    enabled: true
    image: driver
    imagePullSecrets: []
    licensingConfig:
      configMapName: ''
      nlsEnabled: false
    repository: nvcr.io/nvidia
    **version: 525.105.17**

... which isn't compatible with our GPU node RHEL version 8.10 (Ootpa). 

### Relevant log output

```shell
GPU Operator goes into error status, and the nvidia-driver-daemonset-????? pod doesn't start with an error like this: 

Failed to pull image "nvcr.io/nvidia/driver:525.105.17-rhel8.9": rpc error: code = Unknown desc = reading manifest 525.105.17-rhel8.9 in nvcr.io/nvidia/driver: manifest unknown: manifest unknown.
```

**Comment by SalaryTheft** [comment]
Possible duplicate of #1384, which is reported 3 months ago and still not fixed.
You have to manually define the driver version with `GPU_DRIVER_VERSION` variable.

**Comment by durera** [comment]
This was fixed by PR #1558
